### PR TITLE
Improve manager review dashboard page

### PR DESCRIPTION
**Pull request by c0m1t**
Improve the existing table by adding sort and filters and new fields which can be use to track the status and overall rating. Also clicking on each row will open a drawer which displays more details about each review.

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -193,9 +193,13 @@ type Query {
 type RoundNode implements Node {
   title: String!
 
+<<<<<<< HEAD
```
Comment (smmoosavi) [inline schema.graphql]: این‌ها نیاز به اصلاح داره

File: src/locales/fa/messages.po [hunk 2]
```diff
@@ -343,6 +376,10 @@ msgstr "مرورکننده‌ها"
 msgid "Role Expertise and Self Development"
 msgstr "مرجعیت تخصصی"
 
+#: src/pages/manager-review-dashboard/ManagerReviewDashboardTable.tsx:107
+msgid "Rows per page:"
```
Comment (smmoosavi) [inline src/locales/fa/messages.po:380]: متن دو نقطه داره ترجمه نداره

File: src/pages/manager-review-dashboard/useFilters.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      return row.user.includes(filters.user);
+    }
+    return true;
+  };
+
+  const filterStatus = (row: Row) => {
+    if (filters.status === 'todo') {
+      return row.behavioralCompetencies === 0 && row.achievements === 0;
+    }
+    if (filters.status === 'doing') {
```
Comment (smmoosavi) [inline src/pages/manager-review-dashboard/useFilters.ts:32]: نمی‌شه گفت اگه TODO یا DONE نبود پس doing هستش؟

Comment (smmoosavi) [reply src/pages/manager-review-dashboard/useFilters.ts:32]: الببته اگه all بود همواره باید true برگردونه

File: src/pages/manager-review-dashboard/useFilters.ts [hunk 4]
```diff
@@ -33,14 +33,16 @@ export const useFilters = () => {
...
     if (filters.status === 'todo') {
       return row.behavioralCompetencies === 0 && row.achievements === 0;
     }
     if (filters.status === 'doing') {
-      return !(
-        (row.behavioralCompetencies === 100 && row.achievements === 100) ||
-        (row.behavioralCompetencies === 0 && row.achievements === 0)
-      );
+      const done = row.behavioralCompetencies === 100 && row.achievements === 100;
+      const todo = row.behavioralCompetencies === 0 && row.achievements === 0;
```
Comment (c0m1t) [inline src/pages/manager-review-dashboard/useFilters.ts:44]: Maybe we could use the `done` and `todo` variables defined here in 
`filters.status === 'todo'`
and 
filters.status === 'done'

**Pull request merged**